### ISSUE TITLE
Ensure that `map Q gq` in defaults.vim behaves as expected

### ISSUE THREAD

**Issue by Whooleg** [vim/vim#9887]
The `map Q gq` command in the defaults.vim file causes unexpected behavior in select mode.

Consider the following scenario in a clean instance of Vim:

```
source $VIMRUNTIME/defaults.vim
norm aThis is a really long line of arbitrary text. It just needs to be long enough to trigger a reformat.
set textwidth=40
" Select the line, switch to select mode, and press 'Q'.
norm V<C-G>Q
```

Instead of replacing the text with 'Q' as expected, the text is reformatted. This is because the `map Q gq` command in defaults.vim is too broad:

https://github.com/vim/vim/blob/0e1574c4069ded47c170fcd7a47a0df66693a5f4/runtime/defaults.vim#L69

Consider adding `sunmap Q` after this line to ensure that select mode behaves as expected, unless there's some better-suited mapping command that I'm not aware of.

**Comment by brammool** [comment]
Good point.  I suppose it's unusual enough to use Select mode and then type Q that nobody reported this before.
I'll add the "sunmap Q" line.

**Comment by Whooleg** [comment]
Yeah, agreed. I think it's a pretty rare edge case. Thanks for the quick fix Bram! I've really been loving Vim.

**Comment by Whooleg** [comment]
Fixed by commit [1588bc8](https://github.com/vim/vim/commit/1588bc8ebee22f2855f27273fc2234fff370f86c#diff-0adbe06bbbba82d5fd4a111cddd73e979959a2916706c776c810fc72e69c3331R70). Thanks again, Bram!
